tests/test_context.py

```diff
@@ -129,19 +131,28 @@ def test_root_local_override(self):
             with chainerio.open('file://' + __file__) as fp:
                 self.assertEqual(fp.read(), my_script.read().encode("utf-8"))
 
+        chainerio.set_root('posix')
+
     # override with different filesystem
     @unittest.skipIf(shutil.which('hdfs') is None, "HDFS client not installed")
     def test_root_fs_override(self):
         from pyarrow import hdfs
 
-        hdfs_tmpfile = "tmpfile_hdfs"
+        hdfs_tmpdir = "tmpdir_hdfs"
```

Comment:
I don't think we need all these `hdfs_` prefix and `_hdfs` suffix, as these tests are all about HDFS.

Reply:
resolved in 7423eaa